var/spack/repos/builtin/packages/opencascade/package.py

```diff
@@ -18,6 +18,11 @@ class Opencascade(CMakePackage):
 
     maintainers = ["wdconinc"]
 
+    version(
+        "7.6.3",
+        extension="tar.gz",
+        sha256="e7f989d52348c3b3acb7eb4ee001bb5c2eed5250cdcceaa6ae97edc294f2cabd",
+    )
```

Comment:
I get the following from `checksum`:

```suggestion
    version(
        "7.6.3",
        extension="tar.gz",
        sha256="baae5b3a7a38825396fc45ef9d170db406339f5eeec62e21b21036afeda31200",
    )
```

Reply:
You're right. That was the checksum for 7.6.0...